apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -107,7 +107,14 @@ export const sortRows = (data, columnIndexList, orderList) => {
   }
   addRows = orderBy(addRows, columnIndexList, orderList);
   newStudentRows = orderBy(newStudentRows, columnIndexList, orderList);
-  studentRows = orderBy(studentRows, columnIndexList, orderList);
+  // Sort students primarily alphabetically, secondarily by account creation
+  // date which aligns with id # (so that older accounts come first)
+  studentRows = studentRows.sort(function(a, b) {
```

Comment:
You could shorten this function like this: 
`studentRows = studentRows.sort((a, b) => a.name > b.name || a.id - b.id);`
I haven't tested it yet, hope it works :)

Reply:
also nit: `sort` is in-place, so you don't have to reassign the result to `studentRows`.

Reply:
Thank you, will make these changes